FirebaseCombineSwift/Tests/Unit/FirebaseCombine-unit-Bridging-Header.h

```diff
@@ -14,9 +14,25 @@
 
 #import "SharedTestUtilities/FIROptionsMock.h"
 
-#import "FirebaseAuth/FIREmailAuthProvider.h"
+// Firebase Core
+#import "FirebaseCore/Sources/Private/FirebaseCoreInternal.h"
+
+// Firebase Functions
+#import "Functions/FirebaseFunctions/FIRFunctions+Internal.h"
+
+// Firebase Auth
+
+#import <FirebaseAuth/FIREmailAuthProvider.h>
+// ^-- importing FIREmailAuthProvider like this will throw an error from the linter
+
+// #import "FirebaseAuth/Sources/Public/FirebaseAuth/FIREmailAuthProvider.h"
+// ^-- uncommenting this will break the compile
+
+#import "FirebaseAuth/Sources/AuthProvider/Email/FIREmailPasswordAuthCredential.h"
```

Comment:
Trying to resolve this conundrum...

So either I use angle-bracket imports (compiles fine, so that the tests actually see the Combine extensions I made available in my code, BUT has the linter complain about using angle-bracket imports).

Or, I use double-quote imports (which makes the linter happy, but breaks the compile).

@paulb777 any ideas what I might be doing wrong? This happens both in Xcode 12.4 and Xcode 12.5.

Reply:
FIREmailAuthProvider.h is a public header so should be imported with #import <FirebaseAuth/FIREmailAuthProvider.h> outside of the FirebaseAuth module. We should fix the linter...

Reply:
I added a workaround for a similar case at some point #7017. Now we should adjust it again... I think we can try to check for `/Tests/` in the file path.

Reply:
Yep. https://github.com/firebase/firebase-ios-sdk/blob/master/scripts/check_imports.swift#L144 assumes that any non-library code is tests. Bridging headers should also be excluded there.

Reply:
Ok, good to hear I wasn't completely in the wrong :-)